decidim-initiatives/app/helpers/decidim/initiatives/application_helper.rb

```diff
@@ -56,7 +56,7 @@ def scope_children_to_tree(scope)
       def filter_types_values
         types_values = Decidim::InitiativesType.where(organization: current_organization).map do |type|
           TreeNode.new(
-            TreePoint.new(type.id.to_s, type.title[I18n.locale.to_s])
+            TreePoint.new(type.id.to_s, decidim_escape_translated(type.title))
```

Comment:
This isn't done in develop 
```suggestion
            TreePoint.new(type.id.to_s, type.title[I18n.locale.to_s])
```

Reply:
I think this is strongly related to #12674 
If i am applying this code suggestion, we won't be able to have a green pipeline.

Reply:
Ok, if we keep track of this inconsistency between version in that issue I'm ok with that as we need this change for our lord the green color to be happy 💚